atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -329,13 +329,10 @@ def adjust_config_kwargs(self):
         self.config_kwargs['koji_hub'] = koji_map['hub_url']
         self.config_kwargs['koji_root'] = koji_map['root_url']
 
-        odcs_fallback = {
-            'api_url': self.config_kwargs.get('odcs_url'),
-            'insecure': self.config_kwargs.get('odcs_insecure')
-        }
-        odcs_map = get_odcs(self.workflow, odcs_fallback)
-        self.config_kwargs['odcs_url'] = odcs_map['api_url']
-        self.config_kwargs['odcs_insecure'] = odcs_map.get('insecure', False)
+        odcs_map = get_odcs(self.workflow, {})
+        if odcs_map:
+            self.config_kwargs['odcs_url'] = odcs_map['api_url']
```

Comment:
odcs_map.get('api_url')

it won't be defined if it isn't defined in config => get_odcs will return that fallback {}

Reply:
when you do this, you have to modify just 1 line in test_orchestrate_build.py

372: from : expected_kwargs['odcs_insecure'] = None
to: expected_kwargs['odcs_insecure'] = False